File: AppShortcuts/app/src/main/java/github/nisrulz/sample/appshortcuts/DynamicShortcutActivity.java
package github.nisrulz.sample.appshortcuts;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

public class DynamicShortcutActivity extends AppCompatActivity {

    public static final String ACTION = BuildConfig.APPLICATION_ID + ".OPEN_DYNAMIC_SHORTCUT";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_dynamic_shortcut);
    }
}

File: AppShortcuts/app/src/main/java/github/nisrulz/example/appshortcuts/DynamicShortcutActivity.kt
package github.nisrulz.example.appshortcuts

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import github.nisrulz.example.appshortcuts.databinding.ActivityDynamicShortcutBinding

class DynamicShortcutActivity : AppCompatActivity() {

    private lateinit var binding: ActivityDynamicShortcutBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityDynamicShortcutBinding.inflate(layoutInflater)
        binding.apply {
            setContentView(root)
        }
    }

    companion object {
        const val ACTION: String = BuildConfig.APPLICATION_ID + ".OPEN_DYNAMIC_SHORTCUT"
    }
}